src/bin/inx-chronicle/cli.rs

```diff
@@ -170,20 +170,21 @@ impl ClArgs {
                     let exp_ts = time::OffsetDateTime::from_unix_timestamp(claims.exp.unwrap() as _).unwrap();
                     let jwt = auth_helper::jwt::JsonWebToken::new(claims, api_data.secret_key.as_ref())
                         .map_err(crate::api::ApiError::InvalidJwt)?;
-                    println!("Bearer {}", jwt);
-                    println!(
+                    tracing::info!("Bearer {}", jwt);
+                    tracing::info!(
                         "Expires: {} ({})",
                         exp_ts,
                         humantime::format_duration(api_data.jwt_expiration)
                     );
-                    return Ok(true);
+                    return Ok(PostCommand::Exit);
                 }
                 #[cfg(feature = "analytics")]
                 Subcommands::FillAnalytics {
                     start_milestone,
                     end_milestone,
                     num_tasks,
                 } => {
+                    tracing::info!("Connecting to database at bind address `{}`.", config.mongodb.conn_str);
```

Comment:
tbh .. that's not a good idea as the `conn_str` can contain the credentials. You should try to extract the `url` from it.

Reply:
It doesn't contain the credentials. Those are separate config variables.

Reply:
We could parse it into a MongoDB connection String (https://docs.rs/mongodb/2.3.1/mongodb/options/struct.ConnectionString.html) and extract the URL from there. Maybe we create a method for it on `MongoDbConfig`.